core/runner/src/main/java/com/cognifide/aet/runner/processing/steps/CollectionResultsRouter.java

```diff
@@ -122,22 +122,22 @@ private void finishTask() throws JMSException {
     consumer.close();
   }
 
-  private void onSuccess(CollectorResultData collectorResultData, String testName)
-      throws JMSException {
+  private void onSuccess(CollectorResultData collectorResultData, String testName,
+      String correlationId) throws JMSException {
```

Comment:
Try to avoid covering properties. Please rename it to e.g. `patternSuiteCorrelationId`.

Reply:
Fixed.